docs/rest-server/API.md

```diff
@@ -2889,6 +2889,172 @@ Status: 500
 }
 ```
 
+### `GET /api/v2/jobs/:frameworkName/jobAttempts/healthz`
+
+Check if jobAttempts is healthy
+
+*Request*
+
+```json
+GET /api/v2/jobs/:frameworkName/jobAttempts/healthz
+```
+
+*Response if succeeded*
+
+```json
+Status: 200
+OK
+```
+
+*Response if job attempts API not work*
+
+```json
+Status: 501
+Not healthy
+```
+
+### `GET /api/v2/jobs/:frameworkName/jobAttempts`
```

Comment:
previous api endpoint uses kebab case instead of camel case, e.g. `/api/v1/virtual-clusters`,
better to use `job-attempts` to keep consistent?

Reply:
I will change it to ```GET /api/v2/jobs/:frameworkName/job-attempts/:attemptIndex```

It seems like only virtual-clusters use dashes and path params like frameworkName, jobName use camel, however some exceptions:
1. username, like in DELETE /api/v2/user/:username
2. grouplist, like in PUT /api/v2/user/:username/grouplist
3. virtualcluster, like in PUT /api/v2/user/:username/virtualcluster

Reply:
* params are js variables in express and the names aren't in endpoint actually
* no idea about exceptions

Reply:
done in updates